cloud_blobstore/gs.py

```diff
@@ -83,6 +85,15 @@ def _ensure_bucket_loaded(self, bucket: str):
         self.bucket_map[bucket] = bucket_obj
         return bucket_obj
 
+    @CatchTimeouts
```

Comment:
Does the annotation belong on helper methods like this?

Reply:
Agreed, it's redundant. Removing